src/mappings/token.ts

```diff
@@ -23,3 +32,27 @@ export function createToken(tokenAddress: string): void {
     token.save()
   }
 }
+
+export function handleMint(event: Mint): void {
+  handleEvent("Mint(indexed address,uint256)", event, event.address)
```

Comment:
Similarly, `event.address` isn't the associated colony address, it's the token address.

Reply:
Same question as above. In the interest of time, skip handling them ?